File: 1/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{

    for (int i = 0; i < 4; i++)
    {
        printf("?");
    }
    printf("\n");

    // int n = get_int("Tamanho: \n");
    // while (n < 1)
    // {
    //     n = get_int("Tamanho: \n");
    // }

    do
    {
        n = get_int("Tamanho: ");
    }

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: 6/mario.py
def get_int(prompt):
    while True:
        try:
            return int(input(prompt))
        except ValueError:
            pass

def main():
    while True:
        n = get_int("Digite o tamanho da grade: ")
        if n > 0:
            break

    i = 1
    while i != n:
        print("#" * i)
        i += 1

main()

